images/modules/com.redhat.osl.cli.packager/install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2024 Apache Software Foundation (ASF)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+mkdir -p "${KN_WORK_DIR}"
+cd "${KN_WORK_DIR}"
+
+mv  "/tmp/LICENSE" .
+
+wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/amd64/linux/kn-workflow-linux-amd64"
+wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/ppc64le/linux/kn-workflow-linux-ppc64le"
+wget -P "${KN_WORK_DIR}" "${KN_ARTIFACT_URL}/s390x/linux/kn-workflow-linux-s390x"
```

Comment:
I dont think we build images for s390x (system Z), but we do build images for arm (aarch64), shouldn't it be replaced

Reply:
this is not the image, but the client. If we build the kn client for these arch so then we must do the same for the plugin.